src/goto-programs/remove_const_function_pointers.h

```diff
@@ -0,0 +1,96 @@
+/*******************************************************************\
+
+Module: Goto Programs
+
+Author: Thomas Kiley, thomas.kiley@diffblue.com
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_PROGRAMS_REMOVE_CONST_FUNCTION_POINTERS_H
+#define CPROVER_GOTO_PROGRAMS_REMOVE_CONST_FUNCTION_POINTERS_H
+
+#include "goto_model.h"
+#include <util/message.h>
+#include <util/expr.h>
+#include <util/namespace.h>
+
+
+class remove_const_function_pointerst:public messaget
+{
+public:
+  typedef std::set<exprt> functionst;
```

Comment:
I didn't think there was but having changed it, I can think of a reason, though I don't know if it qualifies as good: the tests have to become a lot less precise as the ordering of the functions is no longer deterministic.